department-of-veterans-affairs/abd-vro
# Handle BIE kafka events, pass through the queue and store them in db

## What was the problem?


Associated tickets or Slack threads:
- #1863 
- #1680 

## How does this fix it?[^1]
- Deserialize AVRO data and pass it into Rabbitmq
- Handles string json data for local purposes
  - This will need to be changed to Avro once we have Schema Registry Mocks
- Stores all of the data from BIE kafka messages in db

## How to test this PR
- Merge into develop, deploy it to dev and try out changes


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
